tox.ini

```diff
@@ -115,6 +115,7 @@ deps =
     cython
     pytest-mock
     opentracing
+    pep562
```

Comment:
same here

Reply:
hmm the testa were failing because of the missing import 🤔

Reply:
that is unexpected, was this locally or in CircleCI?

Reply:
CircleCI; locally works fine.